### ISSUE TITLE
Accessibility features

### ISSUE THREAD

**Issue by pavelvlcek** [wbsoft/frescobaldi#835]
Dear developers,
I am a blind computer user. I use screen reader for reading the screen. Frescobaldi can be absolutelly the best editor also for blind users. It has some accessibility features in Linux, but here are some issues. Screen reader Orca reads all dialogues such as preferences etc, but the main edit area has accessibility issues, when I want navigate by line, character, etc, screen reader doesn't speak focus changes. 
If Frescobaldi uses QT 5, can you add accessibility features to your todo list and check it's integration? If it uses QT 4, can you switch to QT 5.5 or greater? In QT 5.5, accessibility features are much better than in qt-at-spi, which must be used in QT 4. I really want to use your software, because if it will be accessible, it will be the first editor, where blind composers can write and read notes / scores and print the result for instrument players. There are no accessibility features for Windows and Mac.
Thank you, 
Pavel Vlček

**Comment by wbsoft** [comment]
The upcoming Frescobaldi 3.x already uses Qt5 so it must be possible to improve the situation. Thanks for reporting!

**Comment by pavelvlcek** [comment]
Dear developers,
I am playing with 3.0 on Arch Linux. The accessibility is much better, but not all elements are correctly accessible. Checkboxes in settings aren't accessible labelled, but it can be maybe QT 5 accessibility bug. Can you check your accessibility integration? Thanks, Pavel.

**Comment by uliska** [comment]
Are you using English or a translation? 

It would also be helpful if you could point us specifically to elements with missing mnemonics.

**Comment by pavelvlcek** [comment]
I am using Czech translation. Orca speaks for example general check box 
and it's status, but orca doesn't speak the name of the checkbox. 
Sometimes, Orca speaks description of the checkbox, but not always.




Dne 20.2.2017 v 20:50 Urs Liska napsal(a):

**Comment by pavelvlcek** [comment]
If you can, please launch Orca with alt super s in Gnome, or alt f2 orca return in Mate gtk-3 and go to edit / preferences. Select general and browse the general settings with tab key.
